### DISCUSSION TITLE
Steckdose/Glühbirne

### DISCUSSION THREAD

**Discussion by mahrens61** [thkl/hap-homematic#637] [Q&A]
Die Homematic Schaltmessdosen werden im Homekit nur als Glühbirne angezeigt. Wie kann ich der HAP beibringen, die Steckdosen als Steckdosen dem Homekit zur Verfügung zu stellen?

**Comment by timwegenerDE** [comment]
Das brauchst du hap nicht beibringen, dass kann die iOS Homeapp selbst umstellen. In den Einstellungen des Geräts: Anzeigen als …

**Reply by mahrens61** [reply]
Diese Einstellung gibt es nicht für die HAP Geräte.

**Reply by mahrens61** [reply]
![IMG_3766](https://github.com/thkl/hap-homematic/assets/13157073/b6000afc-7668-4db3-a1e5-2b67704fed4c)
![IMG_3767](https://github.com/thkl/hap-homematic/assets/13157073/5bfbd6ad-c40d-4235-ab7f-510c3691d1ef)

**Comment by roe1974** [comment]
Welchen Typ hast Du in HAP ausgewählt ? Switch oder Outlet ?

**Comment by gerd2005** [comment]
Hallo, ich habe so eine Steckdose nicht, aber vielleicht kannst du im Plug In einen anderen Service auswählen. Damit kannst du dann in HomeKit in den Geräteeinstellungen auch das Symbol ändern. Ob generell eine Steckdose als Symbol zur Auswahl steht, weiß ich leider auch nicht.

**Comment by blackbasket** [comment]
Kammes sein, dass du den PSM in hap schon als Lampe angelegt hast? Da kann man es in HomeKit nicht ändern. Bei Schaltern ist das kein Problem. Da kannst du dann auch anzeigen als Lampe, Ventilator etc auswählen. Ich meine aber, dass man das auch direkt in hap "richtig" auswählen kann. Ich kann grad leider keine 2. VPN nach Hause auf die RM aufbauen ;)

**Comment by mahrens61** [comment]
Ich habe im HAP jetzt das Device auf Outlet gestellt. Jetz funktioniert es wie es soll. Danke für euren support.
